### ISSUE TITLE
Can anyone reproduce the result for MNIST --> MNIST-M 

### ISSUE THREAD

**Issue by yguooo** [wogong/pytorch-dann#5]
![image](https://user-images.githubusercontent.com/10891654/86670090-e29cc000-bfb9-11ea-9fc1-3d7e74c5131a.png)

What I did is to simply set up my own directory and set momentum = 0, weight_decay = 0 (It is missing from https://github.com/wogong/pytorch-dann/blob/master/experiments/mnist_mnistm.py) However, even after inspecting previous epochs, the best result I could get is around 74%, which is far from the 84% reported in the table. I am wondering if anyone could reproduce this number?

**Comment by yguooo** [comment]
I realize that I need to use the network with dropout and batch normalization, specifically, use `MNISTmodel` instead of `MNISTmodel_plain`. Indeed, the performance of this implementation for mnist --> mnist-m is much better than the original one in the DANN paper. 
![image](https://user-images.githubusercontent.com/10891654/86948048-3fab8980-c112-11ea-8f9c-4eeddc46a564.png)
